src/domain/individualHarbor/IndividualHarborPageContainer.tsx

```diff
@@ -26,7 +27,38 @@ const IndividualHarborPageContainer: React.SFC = () => {
     { variables: { id } }
   );
   const { t, i18n } = useTranslation();
-  const [selectedPier, setSelectedPier] = useState<Pier | null>(null);
+
+  const berthTableGlobalFilter: UseGlobalFiltersOptions<
+    Berth
+  >['globalFilter'] = React.useMemo(
+    () => (rows, _, filterValue) => {
+      return rows.filter(row => {
+        const { number, identifier, length, width, mooringType } = row.original;
+        const numberMatch = String(number) === filterValue;
+        const identifierMatch = String(identifier) === filterValue;
+        const lengthMatch = formatDimension(length, i18n.language)
+          .toLowerCase()
+          .includes(String(filterValue).toLowerCase());
+        const widthMatch = formatDimension(width, i18n.language)
+          .toLowerCase()
+          .includes(String(filterValue).toLowerCase());
+        const mooringTypeMatch = t([
+          `common.mooringTypes.${mooringType}`,
+          mooringType,
+        ])
+          .toLowerCase()
+          .includes(String(filterValue).toLowerCase());
+        return (
+          numberMatch ||
+          identifierMatch ||
+          lengthMatch ||
+          widthMatch ||
+          mooringTypeMatch
+        );
+      });
+    },
+    [t, i18n.language]
+  );
```

Comment:
I find it a little bit error-prone to have a separate function for global filtering because with this approach any change to the `Cell` formatted, has to be repeated it for the global function as well, otherwise, it won't work.
But this is just an opinion.

Reply:
I though of refactoring the cells' formatters to functions that would also be used by the filter. Would that be an acceptable solution?

Reply:
I think that'd be better, but is there any specific reason to not use the `accessor`s' functions?

Reply:
We had a short discussion on slack that doing the formatting in the `accessor` would be bad practice. Now that I've taken another look, we either need to accept that the cell formatting and the global filter are kept in sync, or we move the formatting to the accessor.

Reply:
Sorry, I wasn't clear about what I meant by formatting (changing styles 🙈). My bad!
For example, if we have a number value which needs a unit, `10 m`, and this value and it has to be in red, the accessor takes care of adding the unit and Cell do the styles: 
```JSX
accessor: ({ width }) => `${width} m`,
Cell: ({ cell }) => <span style={{ color: 'red' }}>{cell.value}</span>,
```

Reply:
Ok! In that case, I'll just change the accessors and update the filter accordingly.

Reply:
Thanks!

Reply:
Done

Reply:
Great 👍 
Do we still need to pass `berthTableGlobalFilter`?
I tried to remove it entirely, and the global filter worked as expected.

Reply:
Checked that the global filter defaults to [`text`](https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js#L1) if nothing is specified. The behavior is the same, except that it is not locale aware on the lower case conversion. For the languages we aim to support that is not an issue.

Also explicitly added the filter types to the individual column definitions in the latest commit.

Will keep the changes to the `Table` component, as other tables might want to configure the global filter.

Reply:
🚀